guides/deploy/starter_deploy_cloudfoundry.html

```diff
@@ -37,7 +37,7 @@ <h1 class="header-title">KITURA</h1>
       <h1>DEPLOY YOUR APP USING CLOUD FOUNDRY</h1>
     </div>
 
-<p>This guide will show you how to deploy the Kitura application you created by following our <a href="https://www.kitura.io/guides/gettingstarted">Getting Started</a> guide to a <a href="#cloud-foundry">Cloud Foundry</a> based cloud environment. The guide assumes you already have an environment to deploy to. If not, you can <a href="https://console.bluemix.net/registration/">sign up</a> to IBM Cloud for free and use it to host your Cloud Foundry apps.</p>
+<p>This guide will show you how to deploy the Kitura application you created by following our <a href="guides/gettingstarted.html">Getting Started</a> guide to a <a href="#cloud-foundry">Cloud Foundry</a> based cloud environment. The guide assumes you already have an environment to deploy to. If not, you can <a href="https://console.bluemix.net/registration/">sign up</a> to IBM Cloud for free and use it to host your Cloud Foundry apps.</p>
```

Comment:
These relative links don't work, I think this one needs to be ../gettingstarted.md but you'll need to test it.

Reply:
Correct, when using relative links you need to step back up using `../`. 
In this case you'd want to do `../gettingstarted.html` (we don't have md files currently). 
But as Helen suggested, I'd test that link actually works :)